onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -247,13 +246,14 @@ def calculate_quantization_params(self,quantization_thresholds):
         self._get_input_name_to_nodes(model)
 
         for node in model.graph.node:
-            next_nodes = self._get_next_nodes(model,node)
-            for next_node in next_nodes:
-                node_output_name = next_node.output[0]
-                if node_output_name in quantization_thresholds:
-                    node_thresholds = quantization_thresholds[node_output_name]
-                    node_params = self.calculate_scale_zeropoint(node, next_node, node_thresholds[0], node_thresholds[1])
-                    quantization_params[node_output_name] = node_params
+            for node_output_name in node.output:
+                if node_output_name in self.input_name_to_nodes:
+                    next_nodes = self.input_name_to_nodes[node_output_name]
```

Comment:
Better change next_nodes to some other name. It is wrong concept in previous version.